spec/commands/unreserve_path_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe Commands::UnreservePath do
+  describe "#call" do
+    context "when the path is owned by the app" do
+      it "successfully removes the reservation" do
+        payload = { publishing_app: "foo", base_path: "/bar" }
+        create(:path_reservation, payload)
+        described_class.call(payload)
+        expect(PathReservation.count).to be_zero
+      end
+    end
+
+    context "when the path is not owned by the app" do
+      it "returns an error" do
+        payload = { base_path: "/bar" }
```

Comment:
This test looks a bit like it's testing what happens when a required parameter is missing rather than owned by a different app.

Reply:
Fair enough - I was being minimal. Amended :-).